cli/src/cmd/build.rs

```diff
@@ -153,6 +154,16 @@ fn ensure_maximum_memory_pages(
     Ok(())
 }
 
+/// Strips all custom sections.
+///
+/// Presently all custom sections are not required so they can be stripped safely.
+fn strip_custom_sections(module: &mut Module) {
```

Comment:
haha nice! looks simple

Reply:
This is much better now, but it still doubles the size of master's build. However, it has a reasonable size.